harness/src/main/java/org/jboss/pvt/harness/utils/ClassVersionInspector.java

```diff
@@ -101,4 +97,43 @@ public static boolean checkJarVersion (String filename, ClassVersion version) th
 
         return true;
     }
+
+    /**
+     * Support version scope
+     * @param filename
+     * @param minVersion
+     * @param maxVersion
+     * @return
+     * @throws PVTException
+     */
+    public static boolean checkJarVersion (String filename, ClassVersion minVersion, ClassVersion maxVersion) throws PVTException
+    {
```

Comment:
Agreed that it would be good practices to seperate different fix into separate ticket so we can have better track on things and easier for code review.

@yongyang, Can I suggest to try  to do that seperation as a best pratices?
Personally, I am ok to keep this as it was to move forward.